### PR TITLE
Bugfix/wording

### PR DESCRIPTION
**Pull request by janek26**

### CODE REVIEW

File: packages/extension/e2e/steps/navigateFromAccountToAccountList.ts [hunk 1]
```diff
@@ -3,5 +3,5 @@ import type { Page } from "@playwright/test"
 
 export async function navigateFromAccountToAccountList(page: Page) {
   await page.locator('[aria-label="Show account list"]').click()
-  await expect(page.locator("h6:has-text('My accounts')")).toBeVisible()
```
Comment (bluecco) [inline packages/extension/e2e/steps/navigateFromAccountToAccountList.ts:6]: is correct to have a blank space at the beginning?

File: packages/extension/e2e/steps/newAccount.ts [hunk 2]
```diff
@@ -9,6 +9,6 @@ export async function newAccountWhenEmpty(page: Page) {
 }
 
 export async function newAccount(page: Page) {
-  await expect(page.locator("h6:has-text('My accounts')")).toBeVisible()
+  await expect(page.locator("h6:has-text(' accounts')")).toBeVisible()
```
Comment (bluecco) [inline packages/extension/e2e/steps/newAccount.ts:12]: same

File: packages/extension/e2e/steps/selectAccountFromAccountList.ts [hunk 3]
```diff
@@ -7,6 +7,6 @@ export async function selectAccountFromAccountList(
   page: Page,
   address: string,
 ) {
-  await expect(page.locator("h6:has-text('My accounts')")).toBeVisible()
+  await expect(page.locator("h6:has-text(' accounts')")).toBeVisible()
```
Comment (bluecco) [inline packages/extension/e2e/steps/selectAccountFromAccountList.ts:10]: same

File: packages/extension/e2e/steps/navigateFromAccountToAccountList.ts (hunk 1 above)
Comment (janek26) [reply packages/extension/e2e/steps/navigateFromAccountToAccountList.ts:6]: yep! :D

**Pull request merged**